### PR TITLE
added disabled property to button

### PR DESCRIPTION
**Pull request by Maxyme**
Added a property to set buttons to disabled and reverse.

### CODE REVIEW

File: flexx/ui/widgets/_button.py [hunk 1]
```diff
@@ -115,11 +121,18 @@ def _init_phosphor_and_node(self):
...
-        
+
         @event.connect('text')
         def __text_changed(self, *events):
             self.node.innerHTML = events[-1].new_value
 
+        @event.connect('disabled')
+        def __disabled_changed(self, *events):
+            if events[-1].new_value:
+                self.node.setAttribute("disabled", "disabled")
```
Comment (ivoflipse) [inline flexx/ui/widgets/_button.py]: Setting `"disabled"` to `"disabled"` wouldn't that be a bool or does the value not matter anyway?

Comment (ivoflipse) [inline flexx/ui/widgets/_button.py]:
```
<input disabled />
<input disabled="" />
<input disabled="disabled" />
<input disabled="true" />
<input disabled="false" /> <!-- still disabled! -->
```
Guess the value indeed doesn't matter

Comment (Maxyme) [inline flexx/ui/widgets/_button.py]: it's a funny property:
https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute

**Comment by almarklein** [comment]
Thanks!

File: flexx/ui/widgets/_button.py (hunk 1 above)
Comment (Korijn) [inline flexx/ui/widgets/_button.py]: It's just how HTML works :)

**Pull request merged**